### ISSUE TITLE
parse_ini_file(): errno=13 Permission denied in PHP 7.4

### ISSUE THREAD

**Issue by stralkaj** [dg/ftp-deployment#133]
I found issue, which showed up, while I was running ftp-deployment on PHP 7.4:

`Error: parse_ini_file(): read of 2066 bytes failed with errno=13 Permission denied in C:\Users\neime\Dropbox\wwwroot\ftp-deployment\src\Deployment\CliRunner.php:267`

When I commented condition in CliRunner.php:223, it worked again:

```
/*if (!flock($this->lock = fopen($options['config'], 'r'), LOCK_EX | LOCK_NB)) {
    throw new \Exception('It seems that you are in the middle of another deployment.');
}*/
```

But it isn't ideal solution - it will disable lock.

**Comment by dg** [comment]
Are you using the last release?

**Comment by stralkaj** [comment]
> Are you using the last release?

Yeah, my mistake. I thought that if I run `composer update`, it will update whole project, but it won't. It seems to be fixed in latest version 3.3.

**Comment by cavo789** [comment]
Had the same problem. Thanks @dg for the update and the support
